src/GitHub.VisualStudio/GitHub.VisualStudio.csproj

```diff
@@ -166,13 +166,15 @@
     </EmbeddedResource>
   </ItemGroup>
   <ItemGroup>
+    <Content Include="Resources\logo_128x128.png">
+      <IncludeInVSIX>true</IncludeInVSIX>
+    </Content>
     <Content Include="Resources\preview_200x200.png">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
       <IncludeInVSIX>true</IncludeInVSIX>
     </Content>
     <Content Include="Resources\logo_32x32%402x.png">
```

Comment:
I think we can get rid of `logo_32x32@2x.png` and its associated `VSPackage.resx` entries. Am I correct in thinking that only the one from `source.extension.vsixmanifest` will be used?

https://github.com/github/VisualStudio/blob/3c2d45dfbd0c68ecbc856e2b2fa3658c86f211da/src/GitHub.VisualStudio/VSPackage.resx#L138
https://github.com/github/VisualStudio/blob/3c2d45dfbd0c68ecbc856e2b2fa3658c86f211da/src/GitHub.InlineReviews/VSPackage.resx#L128

Reply:
I can definitely get rid of the 32x32 logo. The reason I didn't was because it might be referenced from somewhere else that I'm not aware of. Not in the extension, but perhaps its URL here on GitHub. 

The resx file can't use .png files to show the logo in Help -> About box. Only .ico files. I've stopped using it because it's not important for the logo to show in the Help  -> About box IMO.

I'll remove it and update the PR

Reply:
Fixed